app/models/concerns/encrypt_field.rb

```diff
@@ -0,0 +1,75 @@
+module EncryptField
```

Comment:
Nitpick, can you call this EncryptString or similar?  "field" has a similar meaning to "attribute" to me, so I was confused for a while with the terminology.